cmd/activator/main.go

```diff
@@ -242,6 +242,7 @@ func main() {
 	ah = &activatorhandler.HealthHandler{HealthCheck: statSink.Status, NextHandler: ah}
 	// NOTE: MetricHandler is being used as the outermost handler for the purpose of measuring the request latency.
 	ah = activatorhandler.NewMetricHandler(revisionInformer.Lister(), reporter, logger, ah)
+	ah = network.NewProbeHandler(ah)
```

Comment:
Is this one going to replace `activatorhandler.ProbeHandler` in future?

And please make the metric handler as the outermost one.

Reply:
No, both probing are different.
See my other comment about the metrics handler.